quickstarts/thumbnails/functions/index.js

```diff
@@ -26,12 +26,12 @@ const fs = require('fs');
 
 // [START generateThumbnail]
 /**
- * When an image is uploaded in the Storage bucket We generate a thumbnail automatically using
- * ImageMagick.
- */
+* When an image is uploaded in the Storage bucket We generate a thumbnail automatically using
+* ImageMagick.
+*/
 // [START generateThumbnailTrigger]
 exports.generateThumbnail = functions.storage.object().onChange(event => {
-// [END generateThumbnailTrigger]
+  // [END generateThumbnailTrigger]
```

Comment:
Remove the spaces, these tags are used for automatically inserting the code in the documentation and this one matches the START tag above so we want to keep the same indentation as the matching tag